pkg/apis/servicecatalog/validation/broker_test.go

```diff
@@ -51,9 +51,11 @@ func TestValidateBroker(t *testing.T) {
 				},
 				Spec: servicecatalog.BrokerSpec{
 					URL: "http://example.com",
-					AuthSecret: &v1.ObjectReference{
-						Namespace: "test-ns",
-						Name:      "test-secret",
+					AuthInfo: servicecatalog.BrokerAuthInfo{
```

Comment:
note that `AuthInfo` cannot be nil because it's not a pointer. `BasicAuthInfo` inside of `AuthInfo` can be nil, though, and that needs a test.